### ISSUE TITLE
Not bale to see logged in user detail

### ISSUE THREAD

**Issue by simarbehl** [enikesha/box-demo#1]
I am not able to view logged in user details, it always return me login details of the user for which i use API client is and API token.

Please suggest what is wrong.

**Comment by enikesha** [comment]
Have you set
BOX_CLIENT_ID
and
BOX_CLIENT_SECRET
in the settings for your box.com application?

**Comment by simarbehl** [comment]
Yes, I have set the BOX_CLIENT_ID and BOX_CLIENT_SECRET. I got these details when i created Application in my box account, under My App (https://app.box.com/developers/services).

Thanks for your help.
